app/views/idv/index.html.slim

```diff
@@ -4,41 +4,37 @@ h1.h3.my0 = t('idv.titles.expectations')
 
 .mb-40p
   .mb-40p
-    | To complete registration with login.gov, we’ll ask you about the following:
+    | To verify your identityv, we’ll ask you to answer a few questions
   h2.mb1.pb-tiny.h5.border-bottom.border-teal Basic information
-  .mb-12p.h6 We’ll start with basic information, like:
+  .mb-12p.h6 We need to know some personal details
   ul.mb0.teal-dots
     li.mb1 Full name
-    li.mb1 Date of birth
     li.mb1 Mailing address
+    li.mb1 Date of birth
     li.mb1 Social Security Number or Individual Tax Identification Number
```

Comment:
`Individual Tax Identification Number` should actually open https://www.irs.gov/individuals/international-taxpayers/taxpayer-identification-numbers-tin#itin in another page. Wasn't sure how to do links if we were just going to internationalize later.